src/libs/Navigation/AppNavigator/ModalStackNavigators.tsx

```diff
@@ -103,7 +103,6 @@ const MoneyRequestModalStackNavigator = createModalStackNavigator<MoneyRequestNa
     [SCREENS.IOU_SEND.ADD_DEBIT_CARD]: () => require('../../../pages/settings/Wallet/AddDebitCardPage').default as React.ComponentType,
     [SCREENS.IOU_SEND.ENABLE_PAYMENTS]: () => require('../../../pages/EnablePayments/EnablePaymentsPage').default as React.ComponentType,
     [SCREENS.MONEY_REQUEST.WAYPOINT]: () => require('../../../pages/iou/MoneyRequestWaypointPage').default as React.ComponentType,
-    [SCREENS.MONEY_REQUEST.DISTANCE]: () => require('../../../pages/iou/NewDistanceRequestPage').default as React.ComponentType,
```

Comment:
I haven't tested this yet, but [in the given example](https://github.com/Expensify/App/pull/33646/files), we don't remove the discussed entry but change it.

Does our case differ?

Reply:
@cubuspl42 It is different case. In this case, I removed SCREENS.MONEY_REQUEST.DISTANCE because we don't use it anymore. 

I marked this PR as draft because I missed this step: `Copy changes since 27 Nov to new components`

I will complete this PR soon. Sorry for confusion 😄

Reply:
That's an important step!

If this can be worked of holding on the https://github.com/Expensify/App/issues/34613, let's do that.